src/helpers.ts

```diff
@@ -0,0 +1,26 @@
+import { Status } from './types/Status';
+import { Todo } from './types/Todo';
+
+export function preperedTodos(
+  todos: Todo[],
+  selectFilter: Status,
+): Todo[] {
+  let filteredTodo = [...todos];
```

Comment:
filter not mutate original data, so i that case you can forgot about cloning original array.

(but i can wrong, mentors know better)

Reply:
Agree with you=)

Reply:
right filter does not mutate array